types/json-schema-faker/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for json-schema-faker 0.5
+// Project: https://github.com/json-schema-faker
```

Comment:
```suggestion
// Project: https://github.com/json-schema-faker/json-schema-faker
```
this is the organization's url, not the actual project url